docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -196,6 +196,12 @@ The configuration parameters which must be set are:
   accessible by all master nodes.
   - Examples: `alluxio.master.journal.folder=hdfs://1.2.3.4:9000/alluxio/journal/`
 
+For clusters with large namespaces, increased CPU overhead on leader could cause delays on Zookeeper client heartbeats. 
+For this reason, we recommend setting Zookeeper client session timeout to at least 2 minutes on large clusters with namespace
+size more than several hundred millions of files. (Zookeeper server's tick time must also be configured as such to allow
+this timeout.)
```

Comment:
Should we link relevant zookeeper documentation, or the specific property name?

Reply:
it's the tickTime. Updated with more info.